README.md

```diff
@@ -1,12 +1,67 @@
+| [@cosmjs/sdk38](packages/sdk38)       | A client library for the Cosmos SDK 0.38                                                                                                                                                                                                 | [![npm version](https://img.shields.io/npm/v/@cosmjs/sdk38.svg)](https://www.npmjs.com/package/@cosmjs/sdk38)       |
+| [@cosmjs/faucet](packages/faucet)     | A faucet application for node.js                                                                                                                                                                                                         | [![npm version](https://img.shields.io/npm/v/@cosmjs/faucet.svg)](https://www.npmjs.com/package/@cosmjs/faucet)     |
+| [@cosmjs/cosmwasm](packages/cosmwasm) | Client for chains with the CosmWasm module enabled                                                                                                                                                                                       | [![npm version](https://img.shields.io/npm/v/@cosmjs/cosmwasm.svg)](https://www.npmjs.com/package/@cosmjs/cosmwasm) |
+| [@cosmjs/crypto](packages/crypto)     | Cryptography for blockchain projects, e.g. hashing (SHA-2, Keccak256, Ripemd160), signing (secp256k1, ed25519), HD key derivation (BIPO39, SLIP-0010), KDFs and symmetric encryption for key storage (PBKDF2, Argon2, XChaCha20Poly1305) | [![npm version](https://img.shields.io/npm/v/@cosmjs/crypto.svg)](https://www.npmjs.com/package/@cosmjs/crypto)     |
+| [@cosmjs/encoding](packages/encoding) | Encoding helpers for blockchain projects                                                                                                                                                                                                 | [![npm version](https://img.shields.io/npm/v/@cosmjs/encoding.svg)](https://www.npmjs.com/package/@cosmjs/encoding) |
+| [@cosmjs/math](packages/math)         | Safe integers; decimals for handling financial amounts                                                                                                                                                                                   | [![npm version](https://img.shields.io/npm/v/@cosmjs/math.svg)](https://www.npmjs.com/package/@cosmjs/math)         |
+
+### Modularity
+
+We're pretty proud of the modularity and a clean dependency tree in this
+monorepo. This ensures software quality on our side and lets users pick exactly
+what they need. Here you see how everything fits together (every item is a npm
+package; right depends on left):
+
+![CosmJS dependency tree](docs/cosmjs-tree.png)
+
+<!--
```

Comment:
Is this a GitHub feature?

Reply:
A feature of markdown is and has always been the ability to embed HTML: https://daringfireball.net/projects/markdown/syntax#html

Reply:
Ah, I meant the content of the comment, not the comment itself. I thought for a minute that GitHub was doing some magic and generating an image based on your command.

Reply:
Ah no, but it would be nice if they would. We could have a bot for rendering and checking-in the images.